build-tools/packages/build-cli/src/repoConfig.ts

```diff
@@ -3,7 +3,28 @@
  * Licensed under the MIT License.
  */
 
+import minimatch from "minimatch";
+import { ReleaseGroup, ReleasePackage } from "./releaseGroups";
+
+// Mapping of branch to a list of release groups/packages that should run policy by default.
+// TODO: This should be configured in the fluid-build config, like type test defaults are.
+const defaults = {
```

Comment:
Map like objects are an anti-pattern in most cases. Since I think this is only iterated over, I think its ok, but if someone did try and optimize lookup to look up branches by name in here, then someone provided a branch named __proto__, or "toString" or such it would likely be mishandled (possibly leading to a prototype pollution bug, which is one of the most common sources of security issues in JS). It would be better to make this an array or a map to reduce the risk of such issues in future versions of this code.